rest_framework_gis/serializers.py

```diff
@@ -132,7 +133,12 @@ def to_representation(self, instance):
         # MUST be present in output according to GeoJSON spec
         field = self.fields[self.Meta.geo_field]
         geo_value = field.get_attribute(instance)
-        feature["geometry"] = field.to_representation(geo_value)
+        if isinstance(field, SerializerMethodField):
+            method = getattr(field.parent, field.method_name)
+            geo_value = method(instance)
+            feature["geometry"] = field.to_representation(instance)
```

Comment:
is using `instance` instead of `geo_value` intended?

Reply:
> is using `instance` instead of `geo_value` intended?

yes, because in this case, `SerializerMethodField.to_representation` receives object instance instead of the actual geometry value. 

the problem was that `geo_value` initially stores the object instance (which doesn't provide `.extent` attribute), so here we're making sure to actually get the geometry value instead of the object instance.

Reply:
> since geo_value is not used afterwards. Isn't it?

yes, it's not used afterwards, but what if -later- someone else needed to use it? 
I believe `geo_value` should have the correct/actual GEO value, not an `instance` of the object.

Reply:
I don't understand this. I don't want to introduce changes in a library that is used by thousands of developers lightly. Each change must be justified.

Reply:
@nemesisdesign it's justified. did you try to reproduce this **bug**? 

focus on the type of `geo_value` variable. it should hold a GEO value and not something else, right?

Reply:
@nemesisdesign if you're not convinced by my solution, then please try to reproduce the bug https://github.com/openwisp/django-rest-framework-gis/issues/247 and maybe suggest a proper fix?